client/assets/stylesheets/_videopress-extends.scss

```diff
@@ -0,0 +1,145 @@
+/**
+ * VideoPress common styles
+ */
+
+// Text input field
+%videopress-form-input-text {
```

Comment:
I don't think I've seen the `%` character in front of the class name before. What's that do?

Reply:
That was a TIL moment for John and I too, its https://sass-lang.com/documentation/style-rules/placeholder-selectors